config/core/configmaps/features.yaml

```diff
@@ -22,3 +22,5 @@ metadata:
     knative.dev/config-propagation: original
     knative.dev/config-category: eventing
 data:
+  # ALPHA feature: The delivery-timeout allows you to use the Timeout field in DeliverySpec.
```

Comment:
should we perhaps point to the umbrella issue here? For background?

Reply:
Good idea, lemme do the same on the other exp feature :)